test/run-tests

```diff
@@ -532,7 +556,9 @@ def choose_task(gh, repos, images, config):
             commands = get_comment_commands(gh, owner, repo, number)
 
             for image in images:
-                status_context = f"{config['name']}/{image['name']}"
+                status_context = get_status_context(
+                    config["name"], image["name"], ansible_ver_str
```

Comment:
```suggestion
                    config["name"], image["name"], ansible_version
```

Reply:
fixed in https://github.com/linux-system-roles/test-harness/pull/87/commits/a1f7e36ec876eb927bb6ddcd25ba619b13dc88c0